src/components/collection/Picker.tsx

```diff
@@ -257,6 +259,34 @@ function CollectionPicker({ readOnly = false }: Props) {
                         onBlur={handlers.validateSelection}
                     />
                 )}
+                renderOption={(props, option, { selected }) => {
+                    return (
+                        <li {...props}>
```

Comment:
Two visual oddities about this list:

* The hover style of a list item lingers on the last item that was hovered.

* The checkmark appears to be uncomfortably close to the left edge of the popper in **light mode only**.

Unsure if the former occurs in production (at the time of commenting), but calling both out for awareness.


### Reference

**Dark Mode**

<img width="353" alt="pr_review-492-available_bindings-dark" src="https://user-images.githubusercontent.com/77648584/220720729-5b7057f4-8964-416a-8034-34e8a376567d.png">

<br />

**Light Mode**

<img width="351" alt="pr_review-492-available_bindings-light" src="https://user-images.githubusercontent.com/77648584/220720796-7eb286b0-0d5d-40ce-aeb3-96bbe32813e9.png">

Reply:
The hover style does not linger for me. What browser are you testing on?

Reply:
Solved the icons shifting with a hack/force of the padding left.

Not sure about the hover styling staying there. If we notice it again and can dig into the cause we can fix it later on.